packages/libs/error-stack/src/lib.rs

```diff
@@ -78,7 +78,18 @@
 //!     # }; Ok(content)
 //! }
 //! # let report = read_file("test.txt").unwrap_err();
-//! # assert_eq!(report.frames().count(), 1);
+//! #
+//! # #[allow(unused_mut)]
+//! # let mut count = 1;
+//! # #[cfg(all(nightly, feature = "std"))]
+//! # {
+//! #   count += 1;
+//! # }
+//! # #[cfg(feature = "spantrace")]
+//! # {
+//! #   count += 1;
+//! # }
+//! # assert_eq!(report.frames().count(), count);
```

Comment:
The tests are already pretty unreadable in code. I won't block the PR on that but we should move out the actual tests to a tests module. If you chose not to do that, could you add a `// TODO: ...` please?

Reply:
Yes, I'll gladly convert them. Trying to maintain/edit the different examples is quite cumbersome.

Reply:
Deferred to doctest update PR